crates/install-wheel-rs/src/linker.rs

```diff
@@ -25,6 +27,9 @@ use crate::wheel::{
 };
 use crate::{Error, Layout};
 
+#[derive(Debug, Default)]
+pub struct Locks(Arc<Mutex<FxHashMap<PathBuf, Arc<Mutex<()>>>>>);
```

Comment:
Looks like the outer `Arc` isn't necessary here.

This also looks very similar to the structure in `uv-distribution`, wonder if we could reuse it somehow.